roles/kubernetes-apps/metallb/tasks/main.yml

```diff
@@ -39,14 +39,19 @@
     src: "{{ item }}.j2"
     dest: "{{ kube_config_dir }}/{{ item }}"
     mode: 0644
-  with_items: ["metallb.yml", "pools.yaml", "layer2.yaml", "layer3.yaml"]
+  with_items: ["metallb.yaml", "pools.yaml", "layer2.yaml","layer3.yaml"]
   register: "rendering"
   when:
     - "inventory_hostname == groups['kube_control_plane'][0]"
 
-- name: Kubernetes Apps | Create MetalLB resources and replace existing
-  k8s:
-    definition: "{{ lookup('template', 'metallb.yaml') }}"
+- name: Kubernetes Apps | Install and configure MetalLB
```

Comment:
Ok so to rewrite this file I would just keep the task line 70 and remove the wait and this task